### PR TITLE
Add Javadoc to ObservableDirectory

### PR DESCRIPTION
**Pull request by jeannetoh99**

### CODE REVIEW

**Review by jiaweiteo — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/ObservableDirectory.java [hunk 1]
```diff
@@ -39,26 +39,44 @@ public ObservableDirectory(Directory directory) {
...
 
+    /**
+     * Returns the friends in the directory as an ObservableList.
+     */
     public ObservableList<Friend> getObservableFriendList() {
         return this.observableFriendList;
     }
 
+    /**
+     * Sets the observable directory to the observe the specified directory.
```
Comment (lyeyixian) [inline src/main/java/seedu/address/model/ObservableDirectory.java:78]: There is a typo here

**Review by lyeyixian — APPROVED** [review]
LGTM. Just a typo.

File: src/main/java/seedu/address/model/ObservableDirectory.java (hunk 1 above)
Comment (jeannetoh99) [reply src/main/java/seedu/address/model/ObservableDirectory.java]: Okay, thanks!

**Pull request merged**